uaclient/cli.py

```diff
@@ -41,6 +44,29 @@
 
 STATUS_FORMATS = ["tabular", "json", "yaml"]
 
+UA_COLLECT_LOGS_FILE = "ua_logs.tar.gz"
+
+UA_REPO_ENTITLEMENTS = (
```

Comment:
This list should be dynamically generated. We can a new variable on the `__init__`  module of `entitlements`